it/controllers/registration/applicant/SoleTraderIdentificationControllerISpec.scala

```diff
@@ -53,6 +53,23 @@ class SoleTraderIdentificationControllerISpec extends ControllerISpec {
 
         val res: Future[WSResponse] = buildClient("/start-sti-journey").get()
 
+        whenReady(res) { result =>
+          result.status mustBe SEE_OTHER
+          result.headers(LOCATION) must contain(testJourneyUrl)
+        }
+      }
+    }
+    "STI returns a journey ID when user is transactor" must {
```

Comment:
It seems to me that this test is not necessary as it is testing the same thing as the test above except setting the `isTransactor` flag which I think is not the determinant factor for redirecting. Please correct me if I am wrong.